step07.md

```diff
@@ -20,17 +20,17 @@ Let's make the code a bit more structured and organised. The best way to do that
 
 `server.js` will remain in the **root**.
 
-**Create a folder called `src`**
+**Create a folder called `src` in the root**
 
-* create src/handler.js
+* create `handler.js` file in the `src` folder
 * paste handler function and related dependencies into handler.js
 
 
 ## Making your own modules!
 
-Way back in step 2, we mentioned there were three types of module in Node. You are about to make the third type of module - your own one!
+Way back in step 2, we mentioned that there are three types of modules in Node. You are about to make the third type of modules - your own one!
 
-In this context, a module is some code you've written in its own JavaScript file. You can share this code with other files. This is handy, because it means we can split big, messy files into smaller ones.
+In this context, a module is some code you've written it in its own JavaScript file. You can share this code with other files. This is handy, because it means we can split big, messy files into smaller ones.
```

Comment:
The "it" that has been added here doesn't make sense. If the sentence before is confusing, perhaps it could be changed to:

In this context, a module is some code *that* you've written in its own JavaScript file.